packages/data/src/registry.js

```diff
@@ -181,14 +174,23 @@ export function createRegistry( storeConfigs = {}, parent = null ) {
 		if ( typeof config.subscribe !== 'function' ) {
 			throw new TypeError( 'config.subscribe must be a function' );
 		}
+		config.emitter = createEmitter();
 		const currentSubscribe = config.subscribe;
 		config.subscribe = ( listener ) => {
-			return currentSubscribe( () => {
-				if ( isPaused ) {
+			const unsubscribeFromStoreEmitter = config.emitter.subscribe(
```

Comment:
This basically means that we have two observables for each store, I don't think it will have any impact but unfortunately the current APIs force us to do this to support "pausing" store subscriptions.